ibis/backends/dask/execution/util.py

```diff
@@ -158,9 +160,7 @@ def _pandas_dtype_from_dd_scalar(x: dd.core.Scalar):
 
 
 def _coerce_to_dataframe(
-    data: Any,
-    column_names: List[str],
-    types: Optional[List[dt.DataType]] = None,
```

Comment:
@datapythonista The problem here is that if `types` is `None` and `data` is a `Series`, this part of the code will raise an error:
```python
    elif isinstance(data, dd.Series):
        dtypes = map(ibis_dtype_to_pandas, types)
```
Would you prefer to manage that case and raise a specific error?

Reply:
I'm fine removing the optional. This is only used internally in `coerce_to_output` which will always pass types.